ui/desktop/app/templates/application.hbs

```diff
@@ -21,25 +21,28 @@
 
       <header.utilities as |utilities|>
         {{#if this.showWindowActions}}
-          <Rose::Button
-            @iconOnly='flight-icons/svg/x-16'
-            @style='warning'
+          <Hds::Button
+            @icon='x'
+            @size='small'
+            @text={{t 'actions.close'}}
+            @color='critical'
             class='button-window-close'
-            title={{t 'actions.close'}}
             {{on 'click' this.close}}
           />
-          <Rose::Button
-            @iconOnly='flight-icons/svg/square-16'
-            @style='secondary'
+          <Hds::Button
+            @icon='x'
+            @size='small'
+            @color='secondary'
+            @text={{t 'actions.fullscreen'}}
             class='button-window-fullscreen'
```

Comment:
After the refactor, is this class need?

Reply:
We are testing for the class [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/desktop/tests/acceptance/application-test.js#L40) so it may be worth a conversation since this is a ui/desktop change. cc: @DhariniJeeva

Reply:
Great advice, will make the change shortly

Reply:
Notice `button-window-minimize` (line 45) and `button-window-close` (line 29) are as per my understanding the same case 😉

Reply:
Looks like we are going to hold off on migrating btns in our ui/desktop and address this in a separate initiative. I've created the ticket here: https://hashicorp.atlassian.net/browse/ICU-16895